service/src/main/java/bio/terra/tanagra/app/controller/ExportApiController.java

```diff
@@ -169,26 +146,18 @@ public ResponseEntity<ApiInstanceListResult> previewExportInstances(
   @Override
   public ResponseEntity<ApiEntityOutputPreviewList> describeExport(
       String underlayName, ApiExportPreviewRequest body) {
-    accessControlService.throwIfUnauthorized(
-        SpringAuthentication.getCurrentUser(),
-        Permissions.forActions(UNDERLAY, READ),
-        ResourceId.forUnderlay(underlayName));
-    for (String featureSetId : body.getFeatureSets()) {
-      accessControlService.throwIfUnauthorized(
-          SpringAuthentication.getCurrentUser(),
-          Permissions.forActions(FEATURE_SET, READ),
-          ResourceId.forFeatureSet(body.getStudy(), featureSetId));
-    }
+    accessControlService.checkUnderlayAccess(underlayName);
+    accessControlService.checkReadAccess(body.getStudy(), body.getCohorts(), body.getFeatureSets());
```

Comment:
If I'm understanding this correctly, the reason for the refactoring is to check cohorts here, which was previously missing. If that is the case, please also file a task to validate the remaining permissions and add some sort of verification. Either way, please note the change in the PR description.

Reply:
I had meant to add a comment that I missed. 
created https://verily.atlassian.net/browse/BENCH-4502